Source/ARTAuth.m

```diff
@@ -263,9 +263,9 @@ - (void)executeTokenRequest:(ARTTokenRequest *)tokenRequest callback:(void (^)(A
 
 - (void)authorise:(ARTTokenParams *)tokenParams options:(ARTAuthOptions *)authOptions callback:(void (^)(ARTTokenDetails *, NSError *))callback {
     BOOL requestNewToken = NO;
-    ARTAuthOptions *mergedOptions = [self mergeOptions:authOptions];
+    ARTAuthOptions *mergedOptions = authOptions ? authOptions : self.options;
```

Comment:
Name is a bit misleading, perhap just `currentAuthOptions` inline with `currentTokenParams` below?